src/build.ts

```diff
@@ -135,6 +185,11 @@ export async function writeTypes(nitro: Nitro) {
   const routes = [
     "// Generated by nitro",
     "import type { Serialize, Simplify } from 'nitropack'",
+    "import type { EventHandler, HTTPMethod } from 'h3'",
+    "import type { FetchOptions } from 'ofetch'",
+    "type DefaultMethod = HTTPMethod | Lowercase<Exclude<HTTPMethod, 'PATCH'>>",
```

Comment:
`Exclude<HTTPMethod, 'PATCH'> | Lowercase<Exclude<HTTPMethod, 'PATCH'>>` ?

Reply:
I think we still want `PATCH` to be an allowable method.